simple_qadi_demo.py

```
@@ -0,0 +1,48 @@
+    print(f"🤖 Agent Types: {result.agent_types}")
+    print(f"💡 Total Ideas Generated: {len(result.synthesized_ideas)}")
+    print("\n" + "=" * 50)
+    
+    # Show ideas from each phase
+    for phase_name, phase_result in result.phases.items():
+        if phase_result and phase_result.generated_ideas:
+            print(f"\n🔸 {phase_name.upper()} PHASE:")
+            for i, idea in enumerate(phase_result.generated_ideas, 1):
+                print(f"  {i}. {idea.content}")
+    
+    print(f"\n🎉 QADI cycle completed successfully!")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)